File: breakfast/breakfast.java/Palindrome.java
public class Palindrome{
	public static void main(String [] args){
	
int number = 121;
int store = number;

String reversed = "";

	while(number > 0){
	
	int dividedNumber = number % 10;
	reversed += dividedNumber;
	number /= 10;

	}

	int convertedNumber = Integer.parseInt(reversed);

	if(store == convertedNumber){
		System.out.print("It is a palindrome");
	}
	else{
	System.out.print("it is not a palindrome");
	}

}
}

File: breakfast/breakfast-javascript/Palindrome.js
let number = 121;
let store = number;

let reversed = "";

	while(number > 0){
	
	let dividedNumber = number % 10;
	reversed += dividedNumber;
	number = Math.trunc(number / 10);

	}

	let convertedNumber = Number(reversed);

	if(store == convertedNumber){
		System.out.print("It is a palindrome");
	}
	else{
	System.out.print("it is not a palindrome");
	}

